src/TodoListItem.module.css

```diff
@@ -0,0 +1,104 @@
+:root {
+  --background-color: #cde9ff;
+  --secondary-color: #e4f3ff;
+  --main-font: "Courier New", Courier, monospace;
+  --main-font-color: #00335ad3;
+  --transition: ease-in-out 125ms;
+  --border-radius: 4px;
+}
+
+.Container {
+  padding: 20px;
+  text-align: center;
+}
+
+.MainHeader {
+  color: var(--main-font-color);
+  font-size: 40px;
+}
+
+.Loading {
+  color: var(--main-font-color);
+  font-family: var(--main-font);
+}
+
+.FormInputField {
+  padding: 12px;
+  margin: 8px;
+  font-size: large;
+  background: var(--secondary-color);
+  border-radius: var(--border-radius);
+  font-family: var(--main-font);
+  border: 1px solid var(--background-color);
+  color: var(--main-font-color);
+}
+
+.ListItem {
+  padding: 10px;
+}
+
+.RemoveButton {
+  font-family: var(--main-font);
+  background-color: var(--secondary-color);
+  border: 1px solid black;
+  border-color: var(--background-color);
+  padding: 15px;
+  border-radius: var(--border-radius);
+  color: var(--main-font-color);
+}
+
+.RemoveButton:hover {
+  cursor: pointer;
+  background-color: grey;
+  color: var(--background-color);
+  transition: var(--transition);
+}
+
+.AddTodoFormButton {
+  font-family: var(--main-font);
+  background-color: var(--secondary-color);
+  border: 1px solid black;
+  border-color: var(--background-color);
+  padding: 15px;
+  border-radius: var(--border-radius);
+  color: var(--main-font-color);
+}
+
+.AddTodoFormButton:hover {
+  cursor: pointer;
+  background-color: grey;
+  color: var(--background-color);
+  transition: var(--transition);
+}
+
+body {
+  background-color: #cde9ff;
+  font-family: var(--main-font);
+  color: var(--main-font-color);
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  color: var(--main-font-color);
+}
+
+form {
+  font-size: 20px;
+}
+
+@media (max-width: 976px) {
```

Comment:
Eventually you'll most likely want to add some more media queries to handle things like tablet and mobile widths too, but I agree not much is needed since your current app doesn't have a ton of width fo it that needs to be adjusted for a narrower page